docs/source/release-notes.html.md.erb

```diff
@@ -0,0 +1,10 @@
+---
+title: Release notes
+weight: 2
+---
+
+# Release Notes
+
+## v1.0 - 13 March 2020
```

Comment:
How can we make this clear that this is the public v1? Or do we care since people shouldn't see the private one ever?

Reply:
I don't think it's needed if people never see the difference. We can add that distinction if we need to document the private api here